### DISCUSSION TITLE
Listing performance/stats from API?

### DISCUSSION THREAD

**Discussion by jvlobo** [etsy/open-api#1304] [Q&A]
Hello there!

I’m a seller, and when I go to the overview tab of one of my listings, I can see the “Performance” section. This section shows metrics like how many views, favorites, sales, and revenue the listing has generated over the last day, 7 days, 30 days, etc.

Is there anything similar available through the Etsy API? The only stat I’ve been able to find is the views property in the listing response, but I haven’t found anything for favorites, sales, or views over a specific period (e.g., the last X days). 🤔

**Comment by anastasiam** [comment]
Hello! I'm also intrested in getting stats data through API by date range. Like the one that you can see in Listing Stats page.

**Reply by jvlobo** [reply]
I reached out to Etsy at developers@etsy.com about a week ago but haven’t received a response. It’s also been two weeks since this issue was posted here without any updates, so I’m unsure about the current activity level of the Etsy API development team.
